rollup.config.js

```diff
@@ -54,6 +51,32 @@ export default {
     commonjs({
       include: 'node_modules/**',
       namedExports: {
+        'node_modules/react/index.js': [
+          'createContext',
+          'createElement',
+          'createRef',
+          'cloneElement',
+          'Children',
+          'Fragment',
+          'forwardRef',
+          'useCallback',
+          'isValidElement',
+          'memo',
+          'useContext',
+          'useState',
+          'useEffect',
+          'useLayoutEffect',
+          'Component',
+          'PureComponent',
+          'useRef',
+          'useReducer',
+          'useMemo',
```

Comment:
It's weird that this is a thing

Reply:
Yeah it's a pain, I don't know why this is necessary.